README.md

```diff
@@ -26,28 +26,28 @@ Docker:
 
 If you're taking an instructor-led workshops then a link will be shared to join the OpenFaaS Slack community. Use the designated channel for the workshop to discuss comments, questions and suggestions.
 
-## [Lab 1 - Prepare for OpenFaaS](./lab1.md)
+## [Lab 1 - Preparing OpenFaaS](./lab1.md)
```

Comment:
Synced the titles with the actual lab markdown.

Reply:
Please revert to the imperative tense. I'm OK if you sync this to the lab too 👍